<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImageCropPreview</class>
 <widget class="QDialog" name="DialogImageCropPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Left Margin:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>original size:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="cropTop">
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="reset">
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="cropRight">
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Top Margin:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="cropLeft">
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QSpinBox" name="step">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Bottom Margin:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="cropBottom">
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>step:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Right Margin</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>640x480</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QLabel" name="ImageLabel">
     <property name="text">
      <string>No Image in clipboard</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogImageCropPreview</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogImageCropPreview</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
